pyvrp/cpp/Individual.h

```diff
@@ -37,6 +39,11 @@ class Individual
      */
     [[nodiscard]] size_t numRoutes() const;
 
+    /**
+     * Number of clients in the solution.
+     */
+    [[nodiscard]] size_t size() const;
```

Comment:
Question: wouldn't `numClients` be a better name and consistent with `numRoutes`? Size is ambiguous, why not measure the size of a solution by the number of routes? (not sure)

Reply:
Yes, that works better. I was thinking about how to detect empty solutions earlier and then figured "oh what about implementing `__len__`", but `individual.num_clients()` works just as well.